gatsby/resolvers/proposal.ts

```diff
@@ -0,0 +1,72 @@
+import axios from "axios"
+
+const BASE_API_URL = "https://forum.threshold.network"
+const PROPOSAL_CATEGORY_URL = `${BASE_API_URL}/c/governance-proposals/7.json`
```

Comment:
just curious about this url. Why does it end in `/7`?

Reply:
It is the category id. Please see https://docs.discourse.org/#operation/listCategoryTopics